components/typography/doc/index.en-US.md

```diff
@@ -12,6 +12,10 @@ Basic text writing, including headings, body text, lists, and more.
 - When need to display title or paragraph contents in Articles/Blogs/Notes.
 - When you need copyable/editable/ellipsis texts.
 
+```ts
```

Comment:
This is not required anymore. Because in ver 9.0.0 and after that, importing from the secondary entry points would be the recommended approach. We already have docs for that somewhere else.